vendor/assets/stylesheets/rolodex/settings/utilities/_layout.sass

```diff
@@ -0,0 +1,51 @@
+// Layout
+// ========================================
+
+// Floats
+
+.pull-left
+  float: left
+
+.pull-right
+  float: right
+
+// Clearfix
+
+.group
+  +group
+
+// Grid
+// ========================================
+
+.grid
+  list-style: none
+  +rem(letter-spacing, -4px)
+
+.opera:-o-prefocus,
```

Comment:
It's a hack to get Opera place nice. :/ Checkout http://bricss.net/post/11230266445/css-hack-to-target-opera for more info.